Submodels/MTSOL_call.py

```
@@ -51,43 +51,94 @@
+                return True
+            time.sleep(0.01)
+        return False
         
 
     def on_modified(self, event):
-        self.on_created(event)
+        if Path(event.src_path).name == self.file_path.name:
+            if self.wait_until_file_free(self.file_path):
+                try:
+                    os.replace(self.file_path, self.destination)
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)